client/src/Components/DataEntry.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  export let footerData = {};
+  const {
+    DESCRIPTION,
+    CONTACT_DETAILS,
+    SUBMIT,
+    SEARCH_STORE
+  } = footerData;
+  const { HEADING, ADDRESS, MOBILE, EMAIL } = CONTACT_DETAILS;
+</script>
+
+<main>
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-md-6">
+      <hr/>
+        <input
+          type="email"
+          class="form-control round-border"
+          placeholder="{SEARCH_STORE}"
+          />
+        <input id="ex1"
+         data-slider-id='ex1Slider'
+         type="text"
+         data-slider-min="0"
+         data-slider-max="5"
+         data-slider-step="1"
+         data-slider-value="3"/>
+
+        <button type="button" class="btn btn-outline-dark round-border">
```

Comment:
wouldn't it make sense to extract this button into it's own component?

Reply:
Mh as you can see I don't have a lot of Frontend experience but I would guess that a single Button is a bit too small for a component?

Reply:
But thats how components should be.. svelte is also following the composition approach like react.